api/filter/accesslog/accesslog.proto

```diff
@@ -32,36 +32,44 @@ message AccessLogCommon {
   // Valid range is (0.0, 1.0].
   double sample_rate = 1 [(validate.rules).double.gt = 0.0, (validate.rules).double.lte = 1.0];
 
-  // This field is the IP and port on which the request from the user was
-  // received.
-  SocketAddress destination_host = 2;
+  // This field is the remote/origin address on which the request from the user was received.
+  // Note: This may not be the physical peer. E.g, if the remote address is inferred from for
+  //       example the x-forwarder-for header, proxy protocol, etc.
```

Comment:
I'm questioning whether this indirection in the HTTP case is worth it, since we are already storing XFF below. Though it is nice to get the "real" downstream address as determined by Envoy. Thoughts?

Reply:
Sorry update here, I know think this is correct. I have it all cleaned up in an Envoy PR to convert all strings to real addresses. I think this should be the inferred trusted downstream. We will also send XFF in headers for comparison.